commands/api/endpoint.go

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"context"
+
+	"github.com/go-kit/kit/endpoint"
+	"github.com/mainflux/mainflux/commands"
+)
+
+func createCommandEndpoint(svc commands.Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(createCommandReq)
+		if err := req.validate(); err != nil {
+			return nil, err
+		}
+		cmd := commands.Command{
+			Command:     req.Command,
+			ChannelID:   req.ChannelID,
+			ExecuteTime: req.ExecuteTime,
+		}
+		id, err := svc.CreateCommand(req.token, cmd)
+		if err != nil {
+			return nil, err
+		}
+		res := createCommandRes{
+			ID:      id,
+			created: true,
+		}
+		return res, nil
+	}
+}
+
+func viewCommandEndpoint(svc commands.Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(viewCommandReq)
+
+		if err := req.validate(); err != nil {
+			return nil, err
+		}
+
+		cmd, err := svc.ViewCommand(req.token, req.id)
+		if err != nil {
+			return nil, err
+		}
+
+		res := viewCommandRes{
+			ID:          cmd.ID,
+			Owner:       cmd.Owner,
+			Name:        cmd.Name,
```

Comment:
Commands do not have particular names, but maybe we can add one. Then we should add `Name` parameter in the CREATE request also.